### PR TITLE
Alternative splicing training: minor modifications

### PR DESCRIPTION
**Pull request by gallardoalba**

### CODE REVIEW

**Comment by hexylena** [comment]
could you also add the news item here as well? https://github.com/galaxyproject/galaxy-hub/pull/2011

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 1]
```diff
@@ -27,15 +27,15 @@ contributions:
...
+Discovered over 40 years ago, alternative splicing (AS) formed a large part of the puzzle explaining how proteomic complexity can be achieved with a limited set of genes  ({% cite Alt1980 %}). The majority of eukaryote genes have multiple transcriptional isoforms, and recent data indicate that each transcript of protein-coding genes contain 11 exons and produce 5.4 mRNAs on average ({% cite Piovesan2016 %}). In humans,  approximately 95% of multi-exon genes show evidence of AS and approximately 60% of genes have at least one alternative transcription start site, some of which exert antagonistic functions ({% cite Carninci2006 %}, {% cite Miura2012 %}).
```
Comment (hexylena) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:30]: fyi we have support for abbreviations if you have more than just AS. https://training.galaxyproject.org/training-material/news/2021/05/25/abbreviations-tag.html

**Comment by gallardoalba** [comment]
> could you also add the news item here as well? [galaxyproject/galaxy-hub#2011](https://github.com/galaxyproject/galaxy-hub/pull/2011) in general we'd love a news items to be added in the same PR as a new tutorial! the hub told us they would consume our news feed in the future, so, that's the model we want to move towards

Sure!

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md (hunk 1 above)
Comment (gallardoalba) [reply topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md]: Done! Indeed there were a large number of abbreviations. Thanks!

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 2]
```diff
@@ -792,13 +797,13 @@ It generates five tabular files with the results of the different statistical an
 
 - Splicing summary: values of the different splicing events
 - Splicing enrichment: results of enrichment statistical analysis for the different splicing events.
-- Consequences summary: values of global usage of isoform switches
+- Consequences summary: values of global usage of {IS}}
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:800]:
```suggestion
- Consequences summary: values of global usage of {IS}
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 3]
```diff
@@ -274,7 +279,7 @@ Before moving to the transcriptome assembly and quantification step, we are goin
 
 ## RNA-seq specific quality control metrics with **RSeQC**
 
-RNA-seq-specific quality control metrics, such as sequencing depth, read distribution and coverage uniformity, are essential to ensure that the RNA-seq data are adequate for transcriptome reconstruction and alternative splicing analysis. For example, the use of RNA-seq with unsaturated sequencing depth gives imprecise estimations  and fails to detect low abundance splice junctions, limiting the precision of many analyses ({% cite Wang2012 %}).
+RNA-seq-specific quality control metrics, such as sequencing depth, read distribution and coverage uniformity, are essential to ensure that the RNA-seq data are adequate for transcriptome reconstruction and a{AS} analysis. For example, the use of RNA-seq with unsaturated sequencing depth gives imprecise estimations  and fails to detect low abundance splice junctions, limiting the precision of many analyses ({% cite Wang2012 %}).
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:282]:
```suggestion
RNA-seq-specific quality control metrics, such as sequencing depth, read distribution and coverage uniformity, are essential to ensure that the RNA-seq data are adequate for transcriptome reconstruction and {AS} analysis. For example, the use of RNA-seq with unsaturated sequencing depth gives imprecise estimations  and fails to detect low abundance splice junctions, limiting the precision of many analyses ({% cite Wang2012 %}).
```

File: topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md [hunk 4]
```diff
@@ -839,15 +844,15 @@ First, we will start analyzing the total number of splicing events (fig. 17).
...
 
-From the figure 17 , it can be hypothesised that some of the alternative splicing events are not equally used. To formally analyze this type of uneven alternative splicing, **IsoformSwithAnalyzeR** computes the fraction of events being gains (as opposed to loss) and perform a statistical analysis of this fraction by using a binomial test (fig. 18).
+From the figure 17 , it can be hypothesised that some of the {AS} events are not equally used. To formally analyze this type of uneven {AS}, **IsoformSwithAnalyzeR** computes the fraction of events being gains (as opposed to loss) and perform a statistical analysis of this fraction by using a binomial test (fig. 18).
 
 > ![Figure 18. Enrichment/depletion in isoform switch with consequences statistical analysis](../../images/differential_isoform/isoformSwitchAnalyzer_splicing_event.png "Comparison of differential splicing events. The fraction (and 95% confidence inter-val) of isoform switches (x-axis) resulting in gain of a specific alternative splice event (indicated by y axis) in the switch from health to cancer. Dashed line indicate no enrichment/depletion. Color indicate if FDR < 0.05 (red) or not (black).")
```
Comment (gallardoalba) [inline topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md:849]:
```suggestion
![Figure 18. Enrichment/depletion in isoform switch with consequences statistical analysis](../../images/differential_isoform/isoformSwitchAnalyzer_splicing_event.png "Comparison of differential splicing events. The fraction (and 95% confidence inter-val) of isoform switches (x-axis) resulting in gain of a specific alternative splice event (indicated by y axis) in the switch from health to cancer. Dashed line indicate no enrichment/depletion. Color indicate if FDR < 0.05 (red) or not (black).")
```

File: news/_posts/2023-05-15-isoform-usage-training.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+---
+title: "New Galaxy training: Genome-wide alternative splicing analysis"
+date: "2023-05-17"
+tease: "GTN hosts a new training for analyzing alternative splicing at genome-wide scale"
+hide_tease: false
+authors: 'Cristóbal Gallardo'
+authors_structured:
+- github: gallardoalba
+- github: hexylena
```
Comment (gallardoalba) [inline news/_posts/2023-05-15-isoform-usage-training.md:9]:
```suggestion
```

**Review by hexylena — DISMISSED** [review]
I've updated it for GTN metadata (which is slightly different than hub) and it all looks good!

**Review by hexylena — APPROVED** [review]

**Pull request merged**